### DISCUSSION TITLE
v7.3.5 Release of PowerShell

### DISCUSSION THREAD

**Discussion by adityapatwardhan** [PowerShell/PowerShell#19865] [General]
### Build and Packaging Improvements

<details>

<summary>

<p>Bump to use .NET 7.0.305</p>

</summary>

<ul>
<li>Update the ThirdPartyNotice (Internal 26372)</li>
<li>Add PoolNames variable group to compliance pipeline (#19408)</li>
<li>Update cgmanifest.json</li>
<li>Update to .NET 7.0.304 (#19807)</li>
<li>Disable SBOM signing for CI and add extra files for packaging tests (#19729)</li>
<li>Increase timeout to make subsystem tests more reliable (#18380)</li>
<li>Increase the timeout when waiting for the event log (#19264)</li>
<li>Implement <code>IDisposable</code> in <code>NamedPipeClient</code> (#18341) (Thanks @xtqqczze!)</li>
<li>Always regenerate files wxs fragment (#19196)</li>
<li>Bump <code>Microsoft.PowerShell.MarkdownRender</code> (#19751)</li>
<li>Delete symbols on Linux as well (#19735)</li>
<li>Add prompt to fix conflict during backport (#19583)</li>
<li>Add backport function to release tools (#19568)</li>
<li>Add an explicit manual stage for changelog update (#19551)</li>
<li>Update the team member list in <code>releaseTools.psm1</code> (#19544)</li>
<li>Verify that packages have license data (#19543)</li>
<li>Fix the regex used for package name check in <code>vPack</code> build (#19511)</li>
<li>Make the vPack PAT library more obvious (#19505)</li>
<li>Update the metadata.json to mark 7.3 releases as latest for stable channel (#19565)</li>
</ul>

</details>

[7.3.5]: https://github.com/PowerShell/PowerShell/compare/v7.3.4...v7.3.5

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - D5F2EC6FCC39431C0F498CE6D5C4577755107F2761DD03178D7DAB182A04815B
- powershell_7.3.5-1.deb_amd64.deb
  - 67DC90155FEA76997645CA7490DB1D33D2AFD7328DF609A2D1C985BE0229C13B
- powershell-7.3.5-1.cm.x86_64.rpm
  - 3570C40B81758E243C412EF0F81C14B797BB9B95AD0D12D1DD6D9D968C1D9F36
- powershell-7.3.5-1.rh.x86_64.rpm
  - AEF79B3E07E3E8177AB2686528825C4346A89B0961E4ED2032719C8ADC64F2D2
- powershell-7.3.5-linux-alpine-x64.tar.gz
  - 694AD368256D3A21052A8C7F60324AEEFB97BA44E009345A69DC4B17126FB4F6
- powershell-7.3.5-linux-arm32.tar.gz
  - C4A893B8F25DA7578D71CC5AC78E90F6317F95F8FC2A17112F1A462844D817F2
- powershell-7.3.5-linux-arm64.tar.gz
  - 96FBEAF8BC57C9BF57B7FD79FAD0CE4475BE0969E785A6657A11C9A1774DC5E1
- powershell-7.3.5-linux-x64.tar.gz
  - 7CE08EE7BED1A882E875738198D7F8EB06861F52EBBD9F8823FF08DA40E7BD21
- powershell-7.3.5-linux-x64-fxdependent.tar.gz
  - 62BC62CD8898FBF52F2327C6114CF180DD986E871B4692812B7CEE67B7ADBCCC
- powershell-7.3.5-osx-arm64.pkg
  - DDFD0476C8D29B87270131E2E3AB2CB3D731F2F8A623F950193F17F1DB1CD0B1
- powershell-7.3.5-osx-arm64.tar.gz
  - 0437BEBE80035218B35904D5090375993CF4FCE9A23C69F8B09416FDDAC035CC
- powershell-7.3.5-osx-x64.pkg
  - F036AA9CB0CF214613D87B3FC825EFACDEDE87B06CA7F6958D9BBDDD958B493D
- powershell-7.3.5-osx-x64.tar.gz
  - 29E94449E73A02A3E36ECBC68D954AA5B35306BDD1C46B9F672492C78C027301
- PowerShell-7.3.5-win-arm32.zip
  - 837B28D73DE33C4B4E7E29697D8D364CB90591F64537BB25239797449DBE6422
- PowerShell-7.3.5-win-arm64.zip
  - E5AFF3D6E1C08527DAD7D4E563A38060060C4F8C70741152CCBA9577C9538F9F
- PowerShell-7.3.5-win-fxdependent.zip
  - A805D108C10F1A5D60C61DB359EB4FE1364D3F8039720F25B2F68412124C8540
- PowerShell-7.3.5-win-fxdependentWinDesktop.zip
  - 16BF05CF9B1287CAC0FF08362F200F7BDAD922CDA2CD78F2BC6F7D30529DE261
- PowerShell-7.3.5-win-x64.msi
  - CF0C8BD757EEC6434A5830352EA6F15ACE237296BB7E4AE778F7707583023AC3
- PowerShell-7.3.5-win-x64.zip
  - C529D644CFEF5A3A7E81F9B68E354EF887F345C7822E3581E9ABD7A627F1A03E
- PowerShell-7.3.5-win-x86.msi
  - E928F7399DDA67C34843C7D278E86D72DDBB151363B821671EB65D19E687CB50
- PowerShell-7.3.5-win-x86.zip
  - 9C269EB3A8ACDA5F263D66842015BA39DC3CECE72B87D482D1DC00D5C5E931AB

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.3.5'>v7.3.5 Release of PowerShell</a>.</em>

**Comment by SpaceTesla** [comment]
Is there a way to upgrade version `7.3.4` to `7.3.5` using command line (PowerShell itself)?

**Reply by c016smith** [reply]
Same , I keep seeing "Upgrade now" as if that's a clickable link or there's been a command shared that would allow that from the existing terminal window. :(
